### ISSUE TITLE
locales are not set during base install

### ISSUE THREAD

**Issue by gubuntu** [AIFDR/inasafe-infrastructure#17]
The Ansible scripted base install for some reason did not configure any locale on the geonode KVMs. This results in, for example, postgresql failing to create a cluster. Please investigate and fix. 

This was the manual fix I did but it should happen automatically during the base install:

```
echo 'LANGUAGE="en_US.UTF-8"' >> /etc/default/locale
echo 'LC_ALL="en_US.UTF-8"' >> /etc/default/locale
```

**Comment by hvisage** [comment]
The install/base used, was a bare minimal, so yes, something like that would be expected as it would've typically been the C (as the standard/default) locale.

Locales could also (and I've done it before when I transferred/moved Postgresql DBs) be forced on the Postgresql cluster creation command lines if it needs to be different from the standard or other installed clusters, like an en_UK vs en_ZA vs the mentioned en_US.

Will add the above to the playbooks.

**Comment by hvisage** [comment]
Refer to https://github.com/AIFDR/inasafe-infrastructure/pull/18

**Comment by timlinux** [comment]
BTW @hvisage normally you can add in your commit message e.g. 

```Fix #17 - set locale to en_US```

Then leave the ticket open, it will get closed when the PR is merged (by virtue of the Fix #<ticket id>). That way the ticket only gets closed when the fix is actually applied.
